common/testdata/romana.sample.yaml

```diff
@@ -14,7 +12,7 @@ services:
     config:
       store: 
         type: sqlite3
-        database: ./ipam.sqlite3
+        database: /tmp/ipam.sqlite3
```

Comment:
This is a default, right? It's for test/demo purposes. In real life you would think where you would put them (or use something more serious than sqlite)